contracts/DAOTreasury.cdc

```diff
@@ -115,18 +114,30 @@ pub contract DAOTreasury {
       emit DepositCollection(collectionID: identifier)
     }
 
+    // Deposit tokens //
+    pub fun depositTokens(identifier: String, vault: @FungibleToken.Vault) {
+      emit DepositTokens(identifier: identifier)
+
+      let vaultRef = (&self.vaults[identifier] as &FungibleToken.Vault?)!
+      vaultRef.deposit(from: <- vault)
+    }
```

Comment:
it looks like we are just changing from ["borrowVault" & "depositVault"] to "depositTokens"? Is it meant to be more readable or what's the purpose of this change?

Reply:
Pinging @jacksonConrad to help here, I'm not sure if this is the right approach
But good catch, depositVault is a really similar to this, so we might just keep one from those two functions

Reply:
good question!  yes, before this PR, the way to deposit tokens into an existing vault was by calling `borrowVaultPublic` and doing `vault.deposit` directly in the transaction.  The downside of this approach is that we cannot emit an `event` for this deposit action when it is done in the transaction.  So the solution is to get rid of `borrowVaultPublic` and replace it with `depositTokens` so we can emit an event, which makes it easier for the Frontend to track.

Reply:
Per discussion from today: Reverted depositVault function to be accessible to the resource owner, and added to create_treasury.cdc initial setup of the flow vault